app/models/service_template.rb

```diff
@@ -435,6 +437,10 @@ def provision_workflow(user, dialog_options = nil, request_options = {})
     end
   end
 
+  def dup
+    super.tap { |obj| obj.update_attributes(:guid => nil) }
```

Comment:
@d-m-u Sorry, I saw this after the fact... Shouldn't this be `super.tap { |obj| obj.guid = nil }` instead?  `dup` doesn't normally write to the database, but this will because of the call to `update_attributes`.

Reply:
Addressed in https://github.com/ManageIQ/manageiq/pull/18490